### DISCUSSION TITLE
Need help setting up live-dvb-tv source as publisher for rtsp-simple-server

### DISCUSSION THREAD

**Discussion by p1ngb4ck** [bluenviron/mediamtx#532] [Questions] [answered]
Hey, me again,

thanks again for your help on my last question (Audio-Track SDP Info), I hope I don`t bother too much ...

Currently I am on a different project, where I am trying to stream free to air live dvb tv via rtsp to enigma2-receiver clients.
What I need is some guidance about how to publish a live dvb tv stream to rtsp-simple-server, without loosing SDP-info for audio tracks -or- some guidance how this could be done with selecting the default/only-mapped audio-track by a specific language. I am not limited to tvheadend in this case, but could use anything that is able to control the tuner directly and open the stream.

Unfortunately, audio-streams cannot be mapped by IDs, as those might change during streaming.

Thanks a bunch for any hints !
Best regards, Oliver

**Comment by aler9** [comment, marked as answer]
> selecting the default/only-mapped audio-track by a specific language.

```
ffmpeg -re -i ... -map 0:v:0 -map 0:m:language:eng -c copy -f rtsp rtsp://localhost:8554/stream
```

> how to publish a live dvb tv stream to rtsp-simple-server, without loosing SDP-info for audio track

I checked both FFmpeg and Gstreamer, and at the moment they don't pass the language information to the RTSP server, regardless of the input file. The language information, should be placed in a track attribute in the SDP:

```
a=lang:<language tag>
```

Therefore, they should be improved first, afterwards we can improve rtsp-simple-server to let that information pass.

**Reply by p1ngb4ck** [reply]
Thanks a lot for your help !!!

[a comment by an automated account is not shown]
